code/game/machinery/requests_console.dm

```diff
@@ -189,7 +196,24 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 			pass = 1
 		if(pass)
 			screen = RCS_SENTPASS
-			message_log += "<B>Message sent to [recipient]</B><BR>[message]"
+			if(recipient in list("Atmospherics","Mechanic","Engineering","Chief Engineer's Desk"))
+				radiochannel = "Engineering"
+			if(recipient in list("Warden","Security","Brig Medbay","Head of Security's Desk"))
+				radiochannel = "Security"
+			if(recipient in list("Bar","Chapel","Kitchen","Hydroponics","Janitorial"))
+				radiochannel = "Service"
+			if(recipient in list("Virology","Chief Medical Officer's Desk","Medbay",))
+				radiochannel = "Medical"
+			if(recipient in list("Blueshield","NT Representative","Head of Personnel's Desk","Captain's Desk","Bridge"))

```

Comment:
going to agree with Sam here.

Make a define for each list that you can check against. Like:
`#define MED_ROLES list("Virology", "Chief Medical Officer's Desk", "Medbay")`

Then, just change those conditionals to use the define. Like:
```if(recipient in MED_ROLES)
   radiochannel = "Medical"```